datamap/bloom/src/main/java/org/apache/carbondata/datamap/bloom/BloomCoarseGrainDataMap.java

```diff
@@ -178,15 +178,9 @@ private String getAncestorTablePath(CarbonTable currentTable) {
     for (BloomQueryModel bloomQueryModel : bloomQueryModels) {
       Set<Blocklet> tempHitBlockletsResult = new HashSet<>();
       LOGGER.debug("prune blocklet for query: " + bloomQueryModel);
-      BloomCacheKeyValue.CacheKey cacheKey = new BloomCacheKeyValue.CacheKey(
-          this.indexPath.toString(), bloomQueryModel.columnName);
-      BloomCacheKeyValue.CacheValue cacheValue = cache.get(cacheKey);
-      List<CarbonBloomFilter> bloomIndexList = cacheValue.getBloomFilters();
-      for (CarbonBloomFilter bloomFilter : bloomIndexList) {
-        if (needShardPrune && !filteredShard.contains(bloomFilter.getShardName())) {
```

Comment:
Why delete the code of filtererShard.contains(bloomFilter.getShardName()), I think this code can reduce time

Reply:
usage of this info is moved to `getBloomFilters`